src/components/LabeledRangeSlider.tsx

```diff
@@ -129,6 +139,14 @@ export default function LabeledRangeSlider(inputProps: LabeledRangeSliderProps):
   stepSize = clamp(stepSize, 0, 1);
   stepSize = setMaxDecimalPrecision(stepSize, 3);
 
+  let marks: undefined | Record<number, ReactNode> = undefined;
+  if (props.marks) {
+    marks = {};
+    props.marks.forEach((value) => {
+      marks![value] = <></>;
```

Comment:
Would recommend using `null` or `undefined` rather than a fragment here (both should be valid values of `ReactNode`)

Reply:
I think Antd checks for null/undefined values here, and ignores them. In this case, we want to override Antd's normal labels for the mark values, and cause it to still render the marks but without any accompanying text. I'll update the comment to explain why it's using a fragment.

Reply:
Gotcha, that makes sense!